source/Calamari.CloudAccounts/AwsEnvironmentGeneration.cs

```diff
@@ -187,7 +189,8 @@ async Task AssumeRole()
                    {
                        RoleArn = assumeRoleArn,
                        RoleSessionName = assumeRoleSession,
-                       ExternalId = string.IsNullOrWhiteSpace(assumeRoleExternalId) ? null : assumeRoleExternalId
+                       ExternalId = string.IsNullOrWhiteSpace(assumeRoleExternalId) ? null : assumeRoleExternalId,
+                       DurationSeconds = int.TryParse(assumeRoleDurationSeconds, out var durationSeconds) ? durationSeconds : 3600
```

Comment:
Minor point, but I would assume passing null here lets the sdk assume the default value instead of hard coding 1 hour.

Reply:
You cant pass null, interestingly (it's an int).
Internally, it's a `int?`, but getting the value returns it as an int, so it ends up as 0.
I've changed it so it's more defensive and doesn't set the property unless we actually try and pass one in so its a bit more future proof.